README.md

```diff
@@ -107,6 +107,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | Project | Description |
 | --- | --- |
 | [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow. |
```

Comment:
```suggestion
| [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow (does not support gpflow >= 2.0 in master). |
```
?

Reply:
sure, though it doesn't support gpflow 1.x either

Reply:
changed it